src/webui/www/private/views/torrentcreator.html

```diff
@@ -0,0 +1,202 @@
+        let progress = false;
+        let timer = null;
+
+        const init = () => {
+            load();
+
+            const contextMenu = new window.qBittorrent.ContextMenu.TorrentCreationTaskContextMenu({
+                targets: "#torrentCreationTasksTableDiv",
+                menu: "torrentCreationTasksTableMenu",
+                actions: {
+                    deleteTask: () => {
+                        const selectedTasks = table.selectedRowsIds();
+                        if (selectedTasks.length === 0)
+                            return;
+
+                        if (confirm("QBT_TR(Are you sure you want to delete selected tasks?)QBT_TR[CONTEXT=TorrentCreator]")) {
+                            selectedTasks.forEach(task => {
+                                fetch("api/v2/torrentcreator/deleteTask", {
+                                    method: "POST",
+                                    cache: "no-store",
+                                    body: new URLSearchParams({
+                                        taskID: task,
+                                    })
+                                }).then((response) => {
+                                    updateData();
+                                });
+                            });
+                        }
+                    },
+                    exportTorrent: exportTorrents,
+                },
+            });
+
+            table.setup("torrentCreationTasksTableDiv", "torrentCreationTasksTableFixedHeaderDiv", contextMenu);
+
+            document.getElementById("addTaskButton").addEventListener("click", (event) => {
+                showCreateTorrentPage();
+            });
+        };
+
+        const unload = () => {
+            clearTimeout(timer);
+            timer = null;
+        };
+
+        const exportTorrents = async () => {
+            const selectedTasks = table.selectedRowsIds();
+            if (selectedTasks.length === 0)
+                return;
+
+            for (const task of selectedTasks) {
+                const url = new URL("api/v2/torrentcreator/torrentFile", window.location);
+                url.search = new URLSearchParams({
+                    taskID: task
+                });
+
+                // download response to file
+                await window.qBittorrent.Misc.downloadFile(url, `${name}.torrent`, "QBT_TR(Unable to export torrent file)QBT_TR[CONTEXT=TorrentCreator]");
+
+                // https://stackoverflow.com/questions/53560991/automatic-file-downloads-limited-to-10-files-on-chrome-browser
+                await window.qBittorrent.Misc.sleep(200);
+            }
+        };
+
+        const load = () => {
+            syncTaskWithInterval(100);
+        };
+
+        const syncTaskWithInterval = (interval) => {
+            if (!progress) {
+                clearTimeout(timer);
+                timer = syncTaskData.delay(interval, null);
```

Comment:
Is `delay` from mootools? If so, then please refrain from using it.
https://github.com/qbittorrent/qBittorrent/tree/627d89813c66bedb4ff51620aefe0fa6ac5101ed/src/webui/www#notices